Added a note to create non-secure APIs. (#2227)
wso2/docs-apim: en/docs/learn/design-api/create-api/create-a-rest-api.md

     | `header`| Contains the case-sensitive names followed by a colon (:) and then by its value that carries additional information with the request, which defines the operating parameters of the transaction. |
     | `cookie` | Operations can also pass parameters in the Cookie header, as `Cookie: name=value`. Multiple cookie parameters are sent in the same header, separated by a semicolon and space.                                                                                            |
     | `body`| An arbitrary amount of data of any type is sent with a POST message.                                                                                                                                |
    
    !!! note

>            If you  have a public API and do not want to subscribe to the application or pass an authorization key to invoke API (without access token), then you can invoke the API successfully by disabling the **security**. 
>            
>            APIs can invoke without using an Access Token. To create non-secure APIs, you have to create an API with the resource having **security: off**.

Reviewer: ```suggestion
           If you want to make it a public API and do not want to pass an authorization key (an access token) to invoke it, turn the **Security** setting **Off** in all the resources of the API.
```

Author: Done with 3ffc38c
